packages/admin/cms-admin/src/builds/BuildEntry.tsx

```diff
@@ -121,8 +120,8 @@ export function BuildEntry(): React.ReactElement {
     const running = data?.builds[0]?.status === "active" || data?.builds[0]?.status === "pending";
 
     return (
-        <DropdownMenuItem buttonIcon={running ? <SsgRunning color="primary" /> : <SsgStandby />} disableArrow>
-            <Content>
+        <AppHeaderDropdown startIcon={running ? <SsgRunning color="primary" /> : <SsgStandby />} dropdownArrow={null}>
+            <Content py={1}>
```

Comment:
Please only use one styling method per element, not a combination of props and styled-component styles.

Reply:
https://github.com/vivid-planet/comet/pull/2541/commits/1a143bf3bfe09b3d000fbf74377f39840085c7d1